internal/spokes/spokes_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestCheckHiddenRefs(t *testing.T) {
-	hiddenRefs := []string{"refs/pull/", "refs/gh/", "refs/__gh__"}
+	hiddenRefs := []string{"refs/pull/", "refs/gh/", "refs/__gh__", "!refs/__github__/svn"}
```

Comment:
```suggestion
	hiddenRefs := []string{"refs/pull/", "refs/gh/", "refs/__gh__", "!refs/__gh__/svn"}
```

I think, otherwise there's nothing to actually unhide.

Reply:
I did this intentionally because I wanted to validate that the actual `!` is working with a reference that's not been previously hidden (that's why I used __github__ instead of __gh).

But looking at the logic again, I think the check is wrong, I have slightly modified it here 8f1d8b6dcca4000c30d9cb4d513d507a00d34262